### PR TITLE
gas estimation calculator on gas page

### PR DESCRIPTION
**Pull request by owocki**
##### Description

This is a gas estimation calculator for the /gas page, so one can see the estimates of how much it costs to do certain actions on gitcoin at today's gas prices.

here is a demo:
screen recording 2018-07-08 at 05 43 pm

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

gas / eth

##### Testing
tested it

##### Refers/Fixes
self

### CODE REVIEW

**Comment by mkosowsk** [comment]
😍

**Comment by owocki** [comment]
any feedback folks? 

i need to get this merged to prove to @mkosowsk that his 250k bounty isnt going to be eaten up by $100k in fees.

**Comment by mkosowsk** [comment]
Seems functional but would appreciate a designer's eye!

File: app/assets/v2/js/base.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ $(document).ready(function() {
 });
 
 $(window).scroll(function() {
-  var scrollPos = $(document).scrollTop();
+  var scrollPos = jQuery(document).scrollTop();
```
Comment (thelostone-mc) [inline app/assets/v2/js/base.js]: Can we stick to `$`?  
That's how it's being used everywhere else

cc @SaptakS

**Comment by thelostone-mc** [comment]
@mkosowsk I'll work that in once this we get a first pass at this ! ^_^

**Comment by mkosowsk** [comment]
@thelostone-mc you are #1! 🙏🏻

File: app/assets/v2/js/base.js (hunk 1 above)
Comment (owocki) [reply app/assets/v2/js/base.js]: $ is not `jQuery` on YGE pages... i'm working to fix this in my other active PR (the YGE refactor).. but until that ships, `$` will cause an error if we leave it as is

**Pull request merged**